code/game/objects/effects/decals/crayon.dm

```diff
@@ -41,3 +41,29 @@
 		cost *= 0.5
 	spraycan.use_charges(user, cost, requires_full = FALSE)
 	return DONT_USE_SPRAYCAN_CHARGES
+
+///Common crayon decals in map.
+/obj/effect/decal/cleanable/crayon/rune4
+	icon_state = "rune4";
```

Comment:
You need to remove all these ``;``'s

Reply:
what do they even do?

Reply:
Nothing good in code, it's only here cause you copied and pasted them from the maps.